tests/cypress/integration/publisher/017-api-policies/01-api-specific-policy.specs.js

```diff
@@ -0,0 +1,105 @@
+
+         //Get the api id
+         cy.location('pathname').then((pathName) => {
+            const pathSegments = pathName.split('/');
+            const uuid = pathSegments[pathSegments.length - 2];
+            cy.visit(`${Utils.getAppOrigin()}/publisher/apis/${uuid}/policies`);
+            //Create API Specific Policy
+            cy.get('[data-testid="add-new-api-specific-policy"]').click();
+            cy.get('#name').type('Add Header sample test');
+            cy.get('input[name="description"]').type('Sample add header policy description');
+            cy.get('#fault-select-check-box').uncheck()
+
+            //upload the policy file
+            cy.get('#upload-policy-file-for-policy').then(function () {
+                const filepath = `api_artifacts/sampleAddHeader.j2`
+                cy.get('input[type="file"]').attachFile(filepath)
+            });
+
+            cy.get('#add-policy-attributes-btn').click();
+            cy.get('[data-testid="add-policy-attribute-name-btn"]').type('headerName');
+            cy.get('[data-testid="add-policy-attribute-display-name-btn"]').type('Header Name');
+            cy.get('#attribute-require-btn').click();
+            //save common policy
+            cy.get('[data-testid="policy-create-save-btn"]').click();
+            cy.wait(2000);
+
+            //View API Specific Policy
+            cy.contains('Add Header sample test').trigger('mouseover');
+            cy.get('[aria-label="view-AddHeadersampletest"]').click();
+            //Download file
+            cy.get('[data-testid="download-policy-file"]').click();
+                const downloadsFolder = Cypress.config('downloadsFolder')
+                const downloadedFilename = `${downloadsFolder}/swagger.yaml`;
+            
+            cy.wait(2000);
+
+            cy.get('[data-testid="done-view-policy-file"]').click();
+
+            //Drag and Drop Policy
+            const dataTransfer = new DataTransfer();
+            cy.contains('Add Header sample test').trigger('dragstart',{
+                dataTransfer
+            });
+            
+            cy.contains('Drag and drop policies here').trigger('drop', {
+            // cy.('[data-testid="drop-policy-zone-request"]').trigger('drop', {
+                dataTransfer
+            });
+            cy.get('#headerName').type('Testing');
+            cy.get('[data-testid="policy-attached-details-save"]').click();
+            cy.get('[data-testid="custom-select-save-button"]').scrollIntoView().click();
+            cy.visit(`${Utils.getAppOrigin()}/publisher/apis/${uuid}/scopes`);
+            cy.visit(`${Utils.getAppOrigin()}/publisher/apis/${uuid}/policies`);
+            cy.wait(2000);
+
+            //Delete API
+            cy.deleteApi(apiName, apiVersion);
+
+         });
+    });
+
+    after(function () {
+        //Delete User
+        cy.visit(`${Utils.getAppOrigin()}/carbon/user/user-mgt.jsp`);
+        cy.deleteUser(publisher);
+        
+    })
+})
```

Comment:
Keep new line at the end of the file

Reply:
Resolved with commit e159e6db870377cf9dc4726b102cf6292273660e